### DISCUSSION TITLE
Will it be possible to resue native loading wheel with ProgressRing?

### DISCUSSION THREAD

**Discussion by takla21** [unoplatform/uno#15816] [Q&A] [answered]
With Xamarin or UWP apps, `ProgressRing` used to utilize native loading wheel. However, upon transitioning to WinUI and NET6+, `ProgressRing` is now using Lottie animation. If I recall, that change was made due to an issue. I'm curious if there are any upcoming plans to restore the choice of utilizing the platform-native loading wheel.

Xamarin/UWP
![IMG_0519](https://github.com/unoplatform/uno/assets/17461593/72f5b473-6ceb-4e34-b383-a21a1ef4db73)

WinUI/NET6+
![IMG_7832](https://github.com/unoplatform/uno/assets/17461593/3a1f3f80-e2d9-47bf-9c19-825050cc881f)

**Comment by jeromelaban** [comment, marked as answer]
See this documentation about the native progress ring: https://platform.uno/docs/articles/features/progressring.html?tabs=uwp

**Reply by Lee31416** [reply]
So we tried and although it works on mobile, on winui it doesn't even with installing uno.winui on the windows head. A workaround for this would be to have a different mobile and windows implementation but it is less than ideal.

**Reply by Lee31416** [reply]
So we found a solution for this, we added a default style where we set the controltemplate to the legacy:ProgressRing only on mobile because it doesn't compile on WinUI. I feel like the documentation was misleading because it said "In WinUI-based Uno Platform apps, this control is in the Uno.UI.Controls.Legacy namespace instead. It is still available as part of Uno Platform 5.x for its support of native styling." which didn't work at all even when installing uno.winui on the windows head. Also the DefaultWux style mentionned in the documentation did nothing on my side.
